tests/13-ieee802154/09-cooja-test-csma-security.csc

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<simconf>
+  <project EXPORT="discard">[APPS_DIR]/mrm</project>
```

Comment:
This file is generated with an old version of Cooja, please open and save the file with the Cooja from latest develop.